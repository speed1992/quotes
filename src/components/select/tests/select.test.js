import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Enzyme, { shallow } from 'enzyme';
import Select from '../select';

Enzyme.configure({ adapter: new Adapter() });

describe('Test Case For Select', () => {
    it('should render component', () => {
        const wrapper = shallow(<Select />)
        expect(wrapper).toMatchSnapshot();
    })
})