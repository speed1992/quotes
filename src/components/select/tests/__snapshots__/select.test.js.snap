// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Case For Select should render component 1`] = `ShallowWrapper {}`;
